### ISSUE TITLE
[Feature Request] log commands send to the cluster

### ISSUE THREAD

**Issue by hbinkle** [harbur/kubernetic#259]
It would be nice to have some king of log showing all the commands send to the cluster when doing any action.

**Comment by dkapanidis** [comment]
Just to clarify, are you looking for a list of all actions that are being performed by the user for historical reasons?

e.g:
- create a deployment x on namespace y
- scale up a statefulset x on namespace y
- port forward service x on namespace y
- cordon node x

Or are you looking for a history of all connections (e.g. list, view, watch resources)?

**Comment by hbinkle** [comment]
right, the first case is important for me. All cation taken on the cluster like create, scale etc.....
as well as stop/start pods, rolling updates etc...

Readonly calls like list, view etc. aren't that of interest, but if it's easier to implement any call (including readonly calls) in a log would do it also. Then we may filter out those calls.

**Comment by dominch** [comment]
I just wanted to make same issue, 
Command, or better command history would help with debuging, ie. when You have metrics empty and would like to test that in terminal You have to guess what command was used and what is answer.
